packages/web/src/components/ui/SidebarCategory/SidebarCategorySearch.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import debounce from 'lodash.debounce';
+import { ScanResultPackageWithMetadata } from '@gradejs-public/public-api/src/clientApiRouter';
+import SidebarCategoryHeader from './SidebarCategoryHeader';
+import Search from '../Search/Search';
+import { SearchText } from 'store/slices/scanDisplayOptions';
+
+type Props = {
+  categoryName: string;
+  packages: ScanResultPackageWithMetadata[] | undefined;
+  searchText: SearchText;
+  handleSearchTextChange: (newSearchText: SearchText) => void;
+};
+
+export default function SidebarCategorySearch({
+  categoryName,
+  searchText,
+  handleSearchTextChange,
+}: Props) {
+  const [searchValue, setSearchValue] = useState<string>('');
+
+  useEffect(() => {
+    setSearchValue(searchText);
+  }, [searchText]);
+
+  const searchChangeHandler = (inputValue: string) => {
```

Comment:
This functions follows `handleSearchTextChange` interface in exact manner, why would we need it here?